templates/redirect_crane.erb

```diff
@@ -0,0 +1,2 @@
+Redirect /v1 https://<%= scope['fqdn'] %>:5000/v1
+Redirect /v2 https://<%= scope['fqdn'] %>:5000/v2
```

Comment:
same comment as in https://github.com/theforeman/puppet-katello/pull/276#pullrequestreview-190796622